osgp/platform/osgp-adapter-ws-core/src/main/webapp/META-INF/context.xml

```diff
@@ -40,4 +40,11 @@
   <Environment name="osgp/AdapterWsCore/log-config"
     value="/etc/osp/osgp-adapter-ws-core-logback.xml" type="java.lang.String"
     override="true" />
-</Context>
\ No newline at end of file
+  
+    <Environment name="osgp/DomainLogging/config"
+    value="/etc/osp/osgp-domain-logging.properties" type="java.lang.String"
+    override="true" />
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)